db/version_set.h

```diff
@@ -374,9 +375,10 @@ class VersionSet {
   {
       bool m_Submitted;     //!< level submitted to hot thread pool
       bool m_Running;       //!< thread actually running compaction
+      uint64_t m_LastCompaction; //!<NowMicros() when last compaction completed
 
       CompactionStatus_s()
-      : m_Submitted(false), m_Running(false)
+      : m_Submitted(false), m_Running(false), m_LastCompaction(Env::Default()->NowMicros())
```

Comment:
The problem with initializing m_LastCompaction via Env::Default()->NowMicros() is that you're mixing and matching environment objects, since m_LastCompaction is set to env_->NowMicros() in SetCompactionDone() above. Normally this assumption is fine, but a different environment object might be used in unit tests. Should this be initialized to 0 here?